tests/db_utils/tests.py

```diff
@@ -83,3 +84,29 @@ def test_load_backend_invalid_name(self):
         with self.assertRaisesMessage(ImproperlyConfigured, msg) as cm:
             load_backend("foo")
         self.assertEqual(str(cm.exception.__cause__), "No module named 'foo'")
+
+
+def connection_is_open():
+    return connection.is_usable()
+
+
+class DatabaseConnectionForkedProcessTests(SimpleTestCase):
+    databases = ["default"]
+
+    @unittest.skipUnless(connection.vendor == "sqlite3", "SQLite are always usable")
+    def test_connections_closed_on_forked_process(self):
+        connection = connections[DEFAULT_DB_ALIAS]
+
```

Comment:
This can be dropped since `connection` is already imported in the context of the module.

```suggestion
```